IsraelHiking.Web/src/application/components/location.component.html

```diff
@@ -21,8 +21,8 @@
            source="location-source"
            [filter]="['==', '$type', 'Point']"
            [layout]="{
-                'icon-image': 'position',
-                'icon-size': 0.95,
+                'icon-image': 'gps-arrow',
```

Comment:
@zstadler this is the part where the icon is added.
The position of the point is coming from the GPS to a geojson source.
This layer is responsible for showing the icon in respect to this point.